app/debrief_svc.py

```diff
@@ -65,20 +53,48 @@ async def build_operation_d3(self, operation_ids):
                                                   type='has_agent'))
         return graph_output
 
+    async def build_attackpath_d3(self, operation_ids):
+        graph_output = dict(nodes=[], links=[])
+        id_store = dict(c2=0)
+        graph_output['nodes'].append(dict(name="C2 Server", type='c2', label='server', id=0, img='server',
+                                          attrs={k: v for k, v in self.get_config().items() if k.startswith('app.')}))
+        agents = await self.data_svc.locate('agents')
+        self._add_agents_to_d3(agents, id_store, graph_output)
+
+        for op_id in operation_ids:
+            operation = (await self.data_svc.locate('operations', match=dict(id=int(op_id))))[0]
+            for link in operation.chain:
+                if link.ability.tactic == 'lateral-movement' and link.status == 0:
```

Comment:
I'm not crazy about hardcoding a tactic in. Let's brainstorm some ways we can solve this lateral movement problem and make things easier.

Reply:
Other options: 
1) just look for the most recently executed techniques before a new agent
2) inspect traits used in abilities looking for 'remote' 
3) extend agents to know IP addresses so that we can match fact values to identify remote commands (or, (3.1) just match on fact values and LM that targets IP addresses)
4) Collect extra data about network traffic from hosts agent live on and use that to infer lateral movement
5) something else? 

I worry (1) is is going to be too broad.  (2) is also subject to the discretion of the ability writer, but I think there's a less solid convention around what fact names to use than att&ck tactics.  (3) might be a good move, but is a heavier lift, requiring changes to core & agents. (3.1) is gonna trip people up. (4) is heavier weight, might seem silly when we do posses tactic level information for commands... 

I like brainstorming, but to push back a little bit:  Why not use tactic level information when we have access to it?